src/main/java/project_16x16/SideScroller.java

```diff
@@ -35,6 +36,22 @@
  * </p>
  */
 public class SideScroller extends PApplet {
+	// " Z: SET FRAMERATE TO 5000 " + "X: SET FRAMERATE TO 20 " + " V: TOGGLE
+	// DEADZONE " + " C: CAMERA TO MOUSE " + " F: HOOK CAMERA TO PLAYER " + " G:
+	// SHAKE SCREEN , POP NOTE " + " P: INCREASE LIFE CAPACITY " + " O: DECREASE
+	// LIFE CAPACITY " + " L: INCREASE LIFE " + " K: DECREASE LIFE " + " F11
+	// FULLSCREEN "
+	private final String Z = " Z: FRAMERATE TO 5000 ";
+	private final String X = " X: FRAMERATE TO 20 ";
+	private final String V = " V: TOGGLE DEADZONE ";
+	private final String C = " C: CAMERA TO MOUSE ";
+	private final String F = " F: HOOK CAM TO PLAYER ";
+	private final String G = " G: SHAKE , POP NOTE ";
+	private final String P = " P: INCREASE LIFE CAP ";
+	private final String O = " O: DECREASE LIFE CAP ";
+	private final String K = " K: DECREASE LIFE ";
+	private final String L = " L: INCREASE LIFE";
+	private final String F11 = " F11: FULLSCREEN ";
```

Comment:
Might be more suitable to move these to Options.class.

Reply:
Aren't these dev related / for debugging ? Hence why I left it in there

Reply:
True, but in my mind it's a more suitable place to store key mappings (even if they are for development only).